src/tests/topp/FeatureFinderIdentification_3_output.featureXML

```diff
@@ -2903,7 +2927,7 @@
 		<feature id="f_17447818880105247092">
 			<position dim="0">1658.35957551813</position>
 			<position dim="1">532.248746583271</position>
-			<intensity>202348</intensity>
+			<intensity>202355</intensity>
```

Comment:
This difference seems to be a newly introduced bug that needs to be addressed.

Reply:
Ok, it seems that this value is within the error tolerance as the current head of ``OpenMS/develop`` produces the same value. The difference is thus not originating from this PR.